test/integration/server.js

```diff
@@ -1155,75 +1155,103 @@ describe('Wallet service', function() {
         });
       });
 
-      it('should create many addresses on simultaneous requests', function(done) {
-        var N = 5;
-        async.map(_.range(N), function(i, cb) {
-          server.createAddress({}, cb);
-        }, function(err, addresses) {
-          addresses.length.should.equal(N);
-          _.each(_.range(N), function(i) {
-            addresses[i].path.should.equal('m/0/' + i);
+      it('should create change address', function(done) {
+        server.createAddress({ isChange: true }, function(err, address) {
+          should.not.exist(err);
+          should.exist(address);
+          address.walletId.should.equal(wallet.id);
+          address.network.should.equal('livenet');
+          address.address.should.equal('1FUzgKcyPJsYwDLUEVJYeE2N3KVaoxTjGS');
+          address.isChange.should.be.true;
+          address.path.should.equal('m/1/0');
+          address.type.should.equal('P2PKH');
+          server.getNotifications({}, function(err, notifications) {
+            should.not.exist(err);
+            var notif = _.find(notifications, {
+              type: 'NewAddress'
+            });
+            should.not.exist(notif);
+            done();
           });
-          // No two identical addresses
-          _.uniq(_.pluck(addresses, 'address')).length.should.equal(N);
-          done();
         });
       });
 
-      it('should fail to create more consecutive addresses with no activity than allowed', function(done) {
-        var MAX_MAIN_ADDRESS_GAP_old = Defaults.MAX_MAIN_ADDRESS_GAP;
-        Defaults.MAX_MAIN_ADDRESS_GAP = 2;
-        helpers.stubAddressActivity([]);
-        async.map(_.range(2), function(i, next) {
-          server.createAddress({}, next);
-        }, function(err, addresses) {
-          addresses.length.should.equal(2);
+      [
+        { isChange: false, branch: 0, thirdAddress: '1GdXraZ1gtoVAvBh49D4hK9xLm6SKgesoE'},
+        { isChange: true, branch: 1, thirdAddress: '13ee82QuHiEz7y5rZjPTsZKfox2rfYeGFv'},
+      ].forEach(function(param) {
```

Comment:
this way we can avoid lots of copy and paste. Such a way of test parameterization looks kind of ugly, feel free to suggest yours